definitions/firefox_desktop.toml

```diff
@@ -1290,6 +1290,11 @@ data_source = "activity_stream_events"
 friendly_name = "Sponsored Tiles Dismissals Count (Positions 1 and 2)"
 description = "Count of sponsored tiles dismissals in the first two positions"
 
+[metrics.separate_search_engine]
```

Comment:
Where was this defined before?

Reply:
I forgot to commit the change removing the definition. Previously, this was defined in [jetstream/defaults/firefox_desktop.toml](https://github.com/mozilla/metric-hub/pull/213/files#diff-9db7038fbcff2586e8a7ca3185f7e6e67eb301454499a7cf59dcf011dcc11e03L59-L65), which is the wrong place for a definition to live.